doc/0001-ADR-Theming.md

```diff
@@ -0,0 +1,58 @@
+# Theming proposal for Open-SCD
+
+Date: 2023-05-16
+
+## Status
+
+Open
```

Comment:
Status can be changed to accepted; if the pull-request is accepted, the ADR is accepted ;)

Reply:
what purpose does the Status section serves then in these? They are always the same value

Reply:
> what purpose does the Status section serves then in these? They are always the same value

It can be:
- "supersedes <adr>"
- "superseded by <adr>"
- "extends <adr>"
- "extended by <adr>"
- etc...

Reply:
> I think we should also add declined ADR's, so we know that it has crossed our minds.

I think that is a good idea. Sometimes we have to know what we already thought of and why it was not a good idea